app/routes/auth.js

```diff
@@ -0,0 +1,36 @@
+const router = require("express").Router();
+const passport = require("passport");
+
+// auth login
+router.get("/login", (req, res) => {
+  res.render("login", { user: req.user });
```

Comment:
Why are u sending user to frontend, when user is not ven logged in?? Also can u write code to redirect to "profile" or "admin" page if user is already logged in basically, he cant view login page unless he logs out.

Reply:
that was made just for testing purposes. there is no need of that particular get router